flybywiresim/a32nx
# feat(ecam): hydraulics page

Fixes #[issue_no]

## Summary of Changes

This is the ECAM SD page for the hydraulics system, written in React. Note this was #3833 (pre-hard fork version).

## Screenshots (if necessary)



![ECAM-HYD_ELEC-PB](https://user-images.githubusercontent.com/5834254/111900467-9a7bdb80-8a2a-11eb-8dc4-77b6fb07c0f8.png)

![ecam-hyd-v4](https://user-images.githubusercontent.com/5834254/111537933-f4974b00-8763-11eb-9475-f188dfb46027.png)


## References


![HYD-page-reference](https://user-images.githubusercontent.com/5834254/110247380-eb3c0080-7f63-11eb-88eb-d7161d74e7c3.png)

![ECAM-HYD-INFLIGHT](https://user-images.githubusercontent.com/5834254/110953652-89064580-833f-11eb-8b30-c165cc1ec4f5.png)





## Additional context



Discord username (if different from GitHub): tricky_dicky[z+0]

## Testing instructions


1. Start both engines on aircraft
2. Go to Hydraulics page on the ECAM SD
3. Disable each engine hydraulic pump in turn. The PTU should transfer pressure from one engine to the other.
4. Disable the PTU and repeat step 3. No pressure transfer should occur
5. Turn off Blue hydraulic pump
6. Test the yellow electric pump by turning off the engine 2 hydraulics pump. Engine 2 hydraulic pressure should be normal


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
